template.rb

```diff
@@ -238,6 +237,7 @@ def get_gh_file_content(filename)
 
 # add RuboCop config
 create_file ".rubocop.yml", get_gh_file_content(".rubocop.yml")
+run "rubocop -A"
```

Comment:
We should run this at the end this file, in case there's anything to fix in any other file added after in the template.

Also, there's one offense that can't be automatically fixed by RuboCop with `rubocop -A`:

```
config/environments/development.rb:19:6: C: Rails/FilePath: Prefer Rails.root.join('path/to').
  if Rails.root.join("tmp", "caching-dev.txt").exist?
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

we should fix that manually too so, after creating the project, the `rubocop` command shows no offenses.

a `gsub_file` replacing `"tmp", "caching-dev.txt"` with `"tmp/caching-dev.txt"` should do it

Reply:
Thanks @arielj , I will update the PR.